Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -257,7 +263,7 @@ internal BlockingRequestHandler(Client client, TimeSpan timeout)
 #endif
 				}
 
-				if (backoff == MAXIMUM_BACKOFF_DURATION && statusCode != (int)HttpStatusCode.OK)
+				if (backoff >= MAXIMUM_BACKOFF_DURATION || statusCode != (int)HttpStatusCode.OK)
```

Comment:
Can you explain this change?

Reply:
`backoff` is changed only after delay where it is multiplied by 2 each time. The initial value is 100, so it will be 200, 400, 800 and so on. `MAXIMUM_BACKOFF_DURATION` is set to 10000 so if statement will never be executed. The second part will work in the next situation: if you get response 400 before `backoff` reached max value then you need to check status code here.

Reply:
Got it - could this be a different PR similar to https://github.com/segmentio/Analytics.NET/pull/83/files#r197556500